test/navigation/NavigatorSpec.scala

```diff
@@ -660,6 +686,60 @@ class NavigatorSpec extends SpecBase with BeforeAndAfterEach {
 
             }
 
+            "NIPHL is not authorised and has NIPHL assesments" in {
+
+              val categoryInfoWithNiphlAssessments = CategorisationInfo(
+                "1234567890",
+                Seq(category1Niphl),
+                Seq.empty,
+                None,
+                1
+              )
+
+              val userAnswers = emptyUserAnswers
+                .set(CategorisationDetailsQuery(testRecordId), categoryInfoWithNiphlAssessments)
+                .success
+                .value
+
+              when(mockCategorisationService.calculateResult(any(), any(), any()))
+                .thenReturn(Category1Scenario)
+
+              navigator.nextPage(
+                CategorisationPreparationPage(testRecordId),
+                NormalMode,
+                userAnswers
+              ) mustBe routes.CategorisationResultController
+                .onPageLoad(testRecordId, Category1Scenario)
+
+            }
+
+            "NIPHL is authorised and has one NIPHL assessments and category 2 assessment with no exemptions" in {
```

Comment:
Possible additional case. Niphl is not authorised and has one Niphl assessment and category 2 no exempt and other category 1 questions

Reply:
Added a new test